### PR TITLE
Typo3 and Neos

### PR DESCRIPTION
**Pull request by jshdoubleu**
Added Typo3 and Neos (or Typo3 Neos) to the CMS section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,6 +287,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Joomla!](http://www.joomla.fr/) - Advanced Content Management System (CMS) `GPL`
   * [KeystoneJS](http://keystonejs.com/) - Node.js CMS & Web Application Platform ([Demo](http://demo.keystonejs.com/), [Source Code](https://github.com/keystonejs/keystone)) `MIT` `Node.js`
   * [MODX](http://modx.com/) - MODX is an advanced content management and publishing platform. The current version is called 'Revolution'. ([Source Code](https://github.com/modxcms/revolution)) `GPLv2` `PHP`
+  * [Neos](https://www.neos.io) - Neos or TYPO3 Neos (for version 1) is a modern, open source MCS based on the Flow framework. ([License](http://typo3.org/index.php?id=782)) `TYPO3 CLA` `PHP`
```
Comment (nodiscc) [inline README.md]: MCS -> CMS

Comment (nodiscc) [inline README.md]: License is in fact GPLv3 (https://typo3.org/index.php?id=782)

Comment (nodiscc) [inline README.md]: Please remove link to the license, and add alink to the source code

File: README.md [hunk 2]
```diff
@@ -294,6 +295,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Redaxscript](http:/redaxscript.com) - Ultra lightweight CMS for MySQL, SQLite and PostgreSQL ([Demo](http://demo.redaxscript.com/demo/login), [Source Code](https://github.com/redaxmedia/redaxscript)) `GPLv3` `PHP`
   * [SilverStripe](https://www.silverstripe.org) - easy to use CMS with powerful MVC framework underlying. ([Demo](http://demo.silverstripe.org/), [Source code](https://github.com/silverstripe)) `BSD` `PHP`
   * [SPIP](http://www.spip.net/fr) - Publication system for the Internet.
+  * [TYPO3](http://typo3.org/) - A powerful and advanced CMS with a large community. ([License](http://typo3.org/index.php?id=782)) `GPLv2` `PHP`
```
Comment (nodiscc) [inline README.md]: Please remove link to the license, and add a link to the source code

**Comment by nodiscc** [comment]
Thank you @jshdoubleu can you fix the points mentioned above?

**Comment by jshdoubleu** [comment]
@nodiscc Yes I'll do

**Comment by nodiscc** [comment]
Thanks!

**Pull request merged**